src/stories/containers/Actors/ActorsContainer.stories.tsx

```diff
@@ -83,14 +54,130 @@ const variantsArgs = [
           linkedIn: '#',
         })
         .build(),
-      itemForList,
-      itemForList,
-      itemForList,
-      itemForList,
       new EcosystemActorBuilder()
         .withId('23')
         .withCode('PH-001')
-        .withName('Pull Up')
+        .withName('Zoenix Labs')
+        .withType('EcosystemActor')
+        .withImage('https://live.staticflickr.com/65535/52808669587_127cc79684_m.jpg')
+        .addCategory('Active Ecosystem Actor')
+        .addScope({
+          id: '1',
+          code: 'SUP',
+          name: 'Support Scope',
+        })
+        .addScope({
+          id: '3',
+          code: 'STA',
+          name: 'Stability Scope',
+        })
+        .withSocials({
+          twitter: '#',
+          forumProfile: '#',
+          forumPlatform: '#',
+          youtube: '#',
+          votingPortal: '#',
+          forumTag: '#',
+          github: '#',
+          discord: '#',
+          website: '#',
+          linkedIn: '#',
+        })
```

Comment:
This object is repeated about ~6 times in this file, maybe you could do something like

```ts
const defaultSocials = {
          twitter: '#',
          forumProfile: '#',
          forumPlatform: '#',
          youtube: '#',
          votingPortal: '#',
          forumTag: '#',
          github: '#',
          discord: '#',
          website: '#',
          linkedIn: '#',
        };
// ...
.withSocials(defaultSocials )
// or maybe:
.withSocials({ ...defaultSocials })
```

This way you avoid to repeat the same block of code every time

Reply:
you are right , done